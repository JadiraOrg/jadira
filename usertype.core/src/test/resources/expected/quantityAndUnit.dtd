<!ELEMENT dataset (
    QUANTITYANDUNIT*)>

<!ELEMENT QUANTITYANDUNIT EMPTY>
<!ATTLIST QUANTITYANDUNIT
    ID CDATA #REQUIRED
    NAME CDATA #IMPLIED
    QUANTITY CDATA #IMPLIED
    UNIT CDATA #IMPLIED
>

