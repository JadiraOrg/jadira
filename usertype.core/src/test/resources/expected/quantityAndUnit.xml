<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dataset SYSTEM "quantityAndUnit.dtd">
<dataset>
  <QUANTITYANDUNIT ID="0" NAME="test_0" QUANTITY="10" UNIT="m"/>
  <QUANTITYANDUNIT ID="1" NAME="test_1"/>
</dataset>
